container/help.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+DOCKER="/usr/bin/docker"
+SELF=$1
+
+echo "OpenSCAP is an auditing tool that utilizes the Extensible Configuration
+Checklist Description Format (XCCDF). XCCDF is a standard way of expressing
+checklist content and defines security checklists.
+"
+
+VERSION=$(${DOCKER} inspect -f '{{ index .Config.Labels "version" }}' ${SELF})
+RELEASE=$(${DOCKER} inspect -f '{{ index .Config.Labels "release" }}' ${SELF})
+if [ -z ${RELEASE} ]; then
+	echo "OpenSCAP image version: ${VERSION}"
+else
+	echo "OpenSCAP image version: ${VERSION}-${RELEASE}"
+fi
+
+echo -e "\n\nSupported datastreams:\n\n"
+DS=$(ls /usr/share/xml/scap/ssg/content/*ds\.xml)
+for d in $DS; do
+	echo "###############################################################################"
+	echo "# $(basename $d)"
+	echo "###############################################################################"
+	oscap info $d
```

Comment:
This will print pretty long output, especially on Fedora, where we ship CentOS and Ubuntu and Scientific and many other datastreams.

Reply:
818 lines on my laptop

Reply:
Yes, but at least user has an easy way how to know what can be scanned. Before, you just had image with no information about datastreams or profiles and it was required to run docker image and know correct path to datastreams if you wanted to know what products and profiles can be used for scanning. So yes, it is long, but `atomic help` opens it for you in `less` and you can search for ds files/profiles easily which is much better and user friendly than running image with docker and searching for it manually.

Reply:
@matusmarhefka I understand your point but I don't think that anybody can read it.